orchestration/OrchestrationService/ConfigurationBuilder.ps1

```diff
@@ -2,62 +2,62 @@ $rootPath = Split-Path -Path $MyInvocation.MyCommand.Definition -Parent
 $helperPath = Join-Path (Join-Path (Join-Path $rootPath -ChildPath '..') -ChildPath 'Common') -ChildPath 'Helper.psm1';
 Import-Module $helperPath;
 
-Class ArchetypeInstanceBuilder {
+Class ConfigurationBuilder {
 
-    $archetypeInstance = $null;
-    $archetypeInstanceName = "";
-    $archetypeDefinitionParentFolder = "";
-    $archetypeDefinitionFileName = "";
+    $configurationInstance = $null;
+    $configurationInstanceName = "";
+    $configurationDefinitionParentFolder = "";
+    $configurationDefinitionFileName = "";
     $fileFunctionResolutionDepthLimit = 20;
 
-    ArchetypeInstanceBuilder([string] $archetypeInstanceName, 
-                             [string] $archetypeDefinitionPath) {
-        # setting the archetype name for use with token parser
-        $this.archetypeInstanceName = $archetypeInstanceName;
+    ConfigurationBuilder([string] $configurationInstanceName, 
+                             [string] $configurationDefinitionPath) {
+        # setting the configuration name for use with token parser
+        $this.configurationInstanceName = $configurationInstanceName;
         
         # check if the path is valid path
-        $filePathExists = Test-Path -Path $archetypeDefinitionPath;
+        $filePathExists = Test-Path -Path $configurationDefinitionPath;
         if($filePathExists) {
-            # archetypeDefinitionParentFolder variable will be used to set 
+            # configurationDefinitionParentFolder variable will be used to set 
             # the base folder for all relative paths. Relative paths are
-            # relative to the archetype definition file's parent folder.
-            $this.archetypeDefinitionParentFolder = (Get-Item $archetypeDefinitionPath).DirectoryName;
-            $this.archetypeDefinitionFileName = (Get-Item $archetypeDefinitionPath).Name;
+            # relative to the configuration definition file's parent folder.
+            $this.configurationDefinitionParentFolder = (Get-Item $configurationDefinitionPath).DirectoryName;
+            $this.configurationDefinitionFileName = (Get-Item $configurationDefinitionPath).Name;
         }   
         else {
-            throw "ArchetypeDefinitionPath value is invalid";
+            throw "ConfigurationDefinitionPath value is invalid";
```

Comment:
Can you please add $configurationDefinitionPath variable as part of the error message, this way we can print out the invalid path as well

Reply:
Updated.